logic/checkUserNumbers.js

```diff
@@ -0,0 +1,27 @@
+const gameRecordsArr = require('../data/gameRecordsArr')
+const strikeAndBall = require('./strikeAndBall')
+const printLog = require('./printLog')
+
+
+
+// 사용자 숫자 입력 및 판별
+function checkUserNumbers(line) 
```

Comment:
개행이 너무 많습니다.
개행은 한개정도로 두는게 맞습니다.

```suggestion
const printLog = require('./printLog')

// 사용자 숫자 입력 및 판별
function checkUserNumbers(line) 
```